apps/backend/src/schema/schema.ts

```diff
@@ -64,4 +68,5 @@ export const formSchema = z.object({
   guardianForm: guardianFormSchema,
   medicalForm: medicalFormSchema,
   adminNotes: adminNotesSchema,
+  read: readSchema.optional(),
```

Comment:
Do we need the readSchema to actually be in the formSchema. Or can it just be an optional boolean, and then readSchema would only be used in the new handler. Seems redundant to have forms pass in the id in two locations, and also less intuitive for the frontend to implement

Reply:
Actually, even the readSchema shouldn't need an id since it gets passed in as a parameter in the request right?

Reply:
Yeah, you're right it does get passed in. I'll change it.

Reply:
Updated.